Cargo.lock

```diff
@@ -1900,17 +1951,30 @@ dependencies = [
  "tiny-keccak 2.0.2",
 ]
 
+[[package]]
+name = "ethcore-transaction"
+version = "0.1.0"
+source = "git+https://github.com/KomodoPlatform/mm2-parity-ethereum.git?branch=fix-pubkey-recover-from-sig#676351596f590a08751eca8ddea5120f513b57a6"
+dependencies = [
+ "ethereum-types",
+ "ethkey 0.3.0 (git+https://github.com/KomodoPlatform/mm2-parity-ethereum.git?branch=fix-pubkey-recover-from-sig)",
+ "keccak-hash",
+ "rlp",
+ "rustc-hex",
+ "unexpected 0.1.0 (git+https://github.com/KomodoPlatform/mm2-parity-ethereum.git?branch=fix-pubkey-recover-from-sig)",
+]
+
 [[package]]
 name = "ethcore-transaction"
 version = "0.1.0"
 source = "git+https://github.com/KomodoPlatform/mm2-parity-ethereum.git?rev=mm2-v2.1.1#d5524212230c4773d01b2527e9b3c422a251e0dc"
 dependencies = [
  "ethereum-types",
- "ethkey",
+ "ethkey 0.3.0 (git+https://github.com/KomodoPlatform/mm2-parity-ethereum.git?rev=mm2-v2.1.1)",
  "keccak-hash",
  "rlp",
  "rustc-hex",
- "unexpected",
+ "unexpected 0.1.0 (git+https://github.com/KomodoPlatform/mm2-parity-ethereum.git?rev=mm2-v2.1.1)",
```

Comment:
Please remove this duplication. This PR https://github.com/KomodoPlatform/mm2-parity-ethereum/pull/2 is not actual anymore. Please revise all deps

Reply:
done.